packages/opencensus-nodejs/test/test-plugin-loader.ts

```diff
@@ -94,7 +94,7 @@ describe('Plugin Loader', () => {
       });
 
       it('should load a plugin and patch the target modules', () => {
-        const pluginLoader = new PluginLoader(log, tracer);
+        const pluginLoader = new PluginLoader(log, tracer, undefined);
```

Comment:
These changes to pass `undefined` would also be unnecessary with it being optional.